File: Integer2Bits/main.cpp
#include<iostream>
#include<conio.h>

using namespace std;
int main()
{
    char choice='N';

	do
	{
        system("cls");

        if (choice == 'Q')
            exit(0);

        int integer = 0,
            binaryNumber[32],
            i = 0;

        cout << "Enter an integer: " << endl;
        cin >> integer;

        while (integer > 0) {
            binaryNumber[i] = integer % 2;
            integer = integer / 2;
            i++;
        }

        cout << endl << "The binary equivalent is: " << endl;
        for (int j = i - 1; j >= 0; j--)
            cout << binaryNumber[j];

        cout << endl << endl << endl << "N: Convert an other number"
            << endl << "Q: Quit the app"
            << endl;

        choice = _getch();
        choice = toupper(choice);
	} while (choice=='N');
}

File: Integer2Bits/main.py
def main():
    while True:
        int_bits = ""
        integer = int(input("Enter an integer:\n"))
        while integer >= 1:
            r = integer%2
            integer = int((integer-r)/2)
            int_bits += str(r)
        print(f"\nThe binary equivalent is:\n{int_bits[::-1]}\n")
        user_choice = str(input("N: Convert an other number\nQ: Quit the app\n"))
        if user_choice.lower() == 'q':
            break

if __name__ == "__main__":
    main()